torch/lib/THD/master_worker/worker/dispatch/Tensor.cpp

```diff
@@ -354,20 +354,6 @@ static void tensorProdall(rpc::RPCMessage& raw_message) {
   }
 }
 
-static void tensorNeg(rpc::RPCMessage& raw_message) {
```

Comment:
I just moved it to another file: https://github.com/apaszke/pytorch-dist/pull/58/files#diff-c0e7ff130c09526e4236f3b315eecf23R344.

Same with `cinv`.